server/server.js

```diff
@@ -129,6 +165,83 @@ app.post('/api/sign-in', async (req, res, next) => {
   }
 });
 
+// User can add to shopping cart
+app.post(
+  '/api/orderItems/:orderItemId',
```

Comment:
You want the `orderId` here, not the `orderItemId`. When they add something to the cart, the database will generate a new orderItemId.